packages/libidn2.rb

```diff
@@ -9,17 +9,17 @@ class Libidn2 < Package
   source_url 'https://ftpmirror.gnu.org/libidn/libidn2-2.3.1.tar.gz'
   source_sha256 '8af684943836b8b53965d5f5b6714ef13c26c91eaa36ce7d242e3d21f5d40f2d'
 
-  binary_url ({
+  binary_url({
     aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libidn2/2.3.1_armv7l/libidn2-2.3.1-chromeos-armv7l.tpxz',
      armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libidn2/2.3.1_armv7l/libidn2-2.3.1-chromeos-armv7l.tpxz',
        i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libidn2/2.3.1_i686/libidn2-2.3.1-chromeos-i686.tpxz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libidn2/2.3.1_x86_64/libidn2-2.3.1-chromeos-x86_64.tpxz',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libidn2/2.3.1_x86_64/libidn2-2.3.1-chromeos-x86_64.tpxz'
   })
-  binary_sha256 ({
-    aarch64: '9533e091572a2fb3aff874794327ffb752967b9952608cbccbd06321a1098323',
-     armv7l: '9533e091572a2fb3aff874794327ffb752967b9952608cbccbd06321a1098323',
-       i686: 'f65ae2734d93e883e32a7602bee7e494d396406addd38d675346d41c5eb34943',
-     x86_64: '3a54e6d6c60865885257e3f1383a1b1d957483a97ef5c0ff400611f58908f28e',
+  binary_sha256({
+    aarch64: 'd5451f73189a53bbdd500b23c781750d688d9b8ee4763eecca8323eace221566',
+     armv7l: 'd5451f73189a53bbdd500b23c781750d688d9b8ee4763eecca8323eace221566',
+       i686: '03df0cdc3848415a7e08e2026eab5a3a50c6943c1980223dac8d0579cdbdc3be',
+     x86_64: '2ade8b8171841b7a911b1802d2dbee0f890537687131e16515a4155c9167187a'
```

Comment:
Is this just because the original sha256sums were wrong?  Normally, we would need to bump the version if that wasn't the case.

Reply:
This is a rebuild I'll bump the version.

Reply:
adjusted